core-customize/tccore/src/com/taloscommerce/core/event/SendOldProductInWishListEvent.java

```diff
@@ -0,0 +1,26 @@
+package com.taloscommerce.core.event;
+
+import com.taloscommerce.core.model.SendOldProductInWishListProcessModel;
+import de.hybris.platform.basecommerce.model.site.BaseSiteModel;
+import de.hybris.platform.commerceservices.event.AbstractCommerceUserEvent;
+import de.hybris.platform.core.model.product.ProductModel;
+
+import java.util.List;
+
+public class SendOldProductInWishListEvent extends AbstractCommerceUserEvent<BaseSiteModel> {
+
+    private List<ProductModel> products;
```

Comment:
I recommend that the product variable be called oldProducts just to have a better reference of what we are saving there